### PR TITLE
Tw/feature/add release notes page

### PR DESCRIPTION
**Pull request by twessling-icas**
adding release notes page.
Seems there is a bug with docutils & sphinx-doc resulting in bulleted & numbered lists not being rendered properly...
Also made the heading of all `Example` occurrences consistent.

### CODE REVIEW

File: src/source/categories.rst [hunk 1]
```diff
@@ -13,12 +13,16 @@ attribute definitions and validation rules for placing ads.
...
+.. list-table::
+ :widths: 70
+ :header-rows: 0
+
+ - * categories form a tree with a single root node
+ - * each category has a unique id
+ - * each category specifies the locales for which it is available
+ - * each ad belongs to one and only one category
+ - * ads can only be placed in leaf categories
+ - * only leaf categories contain attributes
```
Comment (twessling-icas) [inline src/source/categories.rst:25]: made this into a table due to the bug with sphinx & docutils...

File: src/source/examples/post-campaign-v5-example.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.. code-block:: javascript
+
+    POST /campaign
```
Comment (remenska) [inline src/source/examples/post-campaign-v5-example.rst:3]: we want to already announce it? Can we guarantee budgets always there when a campaign is created?

File: src/source/examples/post-campaign-v5-example.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                "limitMicros": "10000000",
+            },
+            "total": {
+                "limitMicros": "UNLIMITED",
+            }
+        }
+    }
+
+    200 OK HTTP/1.1
+    Content-Type: application/sellside.campaign-v5+json; charset=UTF-8
```
Comment (remenska) [inline src/source/examples/post-campaign-v5-example.rst:20]: where do we draw the line with respect to v5-ing API endpoints? :) this jumped quite some versions, I'm just curious.... Is the idea that all related endpoints are v5?

File: src/source/reference/get-user.rst [hunk 4]
```diff
@@ -3,13 +3,46 @@
...
+:ref:`get_user_v4` | :ref:`get_user_v3` | :ref:`get_user_v2` | :ref:`get_user_v1`
+
+This URL returns information of the current user. The ``id`` field contains
+a unique user identifier. The ``options.feed`` field is ``true`` if the user
+is an XML feed customer. The ``options.vanityUrl`` field is ``true`` if the
+value of the :ref:`ad_links_displayUrl` field in the advertisement is shown
+in the search results. The ``hasAds`` field is ``false`` if the user has no ads at all (not even deleted).
+The ``locations`` field contains the user (or webshop) locations.
+The ``isAPIManaged`` field is ``true`` when there is (at least one) API partner
+using RW-scopes for this customer. API partners using RO-scopes only will not
```
Comment (remenska) [inline src/source/reference/get-user.rst]: why don't we use the same lingo as the Scope section `api_rw` (also makes it clear that we don't treat console as API partner :) )

File: src/source/release-notes.rst [hunk 5]
```diff
@@ -0,0 +1,194 @@
+.. index:: Release Notes
+.. _release_notes:
+
+Release Notes
+=============
+
+This section attempts to give a little more insight on what our various releases contain, as compared to previous versions.
```
Comment (remenska) [inline src/source/release-notes.rst]:
```suggestion
This section gives insight on what our current and previous releases contain.
```

File: src/source/release-notes.rst [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+Feb 2023: 'Campaigns & Micros'
+------------------------------
+
+Synopsis
+""""""""
+
+* For bid, budgets & spent values we now report in micros: 1 cent == 10000 micros. These fields will now also be named accordingly, mentioning whether they represent a value in cents or micros. This is also represented in the categories data, where the interval of allowed values is defined. Mind, this does not include values like 'AUTOMATIC' or 'UNLIMITED'.
```
Comment (remenska) [inline src/source/release-notes.rst]: imho the word "bid" has not been introduced before to our API partners, at least not officially?

Comment (remenska) [reply src/source/release-notes.rst]: > Mind, this does not include values like 'AUTOMATIC' or 'UNLIMITED'.
Can you expand on this? I got confused for a second on what _this_ refers to

File: src/source/release-notes.rst [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+* Introduced campaigns, campaign ID not (yet) mandatory while placing ads. Campaigns have their own daily/total budgets. Since currently only 1 campaign is allowed per user, this effectively represents the seller's complete budget.
```
Comment (remenska) [inline src/source/release-notes.rst]: nit: we first speak of "budgets" but then "budget". Maybe rephrase a bit?

File: src/source/release-notes.rst [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+* Moved price and related fields into a struct.
+* removed externalID, use vendorID instead.
```
Comment (remenska) [inline src/source/release-notes.rst:23]: Maybe bundle the below into "removed fields" bullet list?

File: src/source/release-notes.rst [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+* removed currency, 1 country == 1 currency anyway. Monetary values always in local currency.
+* removed pageNumber & suggestedCpcForPageOne: Numbers are unreliable and only work for the leaf category browse action, which by now is only a very small portion of our traffic on any of our marketplaces. Therefore, these numbers are not representative anymore, and we chose to remove them.
```
Comment (remenska) [inline src/source/release-notes.rst]: I would invite here a look from the product owner, to rephrase it a little bit so it's clearer and sounds less bad?

File: src/source/release-notes.rst [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+
+* :ref:`get_campaign` to get list of campaigns for a user. Expected 0 or 1 result.
+* :ref:`get_campaign_id` to fetch the individual campaign.
+* :ref:`post_campaign` to create a campaign. Currently only 1 allowed per seller.
+* :ref:`put_campaign_id` to update the campaign. Currently only status & budgets can be updated.
+* :ref:`put_campaign_id_status` to manage the campaign status (ACTIVE,PAUSED) directly.
+* :ref:`put_campaign_id_budgets` to manage the campaign's budgets directly.
+
+.. note::
+ For the time being, we will create a campaign underwater for users who start placing ads without first having created a campaign.
```
Comment (remenska) [inline src/source/release-notes.rst]:
```suggestion
 For the time being, we will create a campaign underwater for users who start placing ads without first having created a campaign. All subsequent ads will belong to that default campaign.
```

File: src/source/release-notes.rst [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+ Deleting a campaign is currently not allowed.
+
+User
+""""
+
+See :ref:`get_user` for full details.
+
+Start using V4. There are 2 new fields in comparison with V2:
+``hasAds`` (**bool**), whether a user has ads in the system and 
+``isApiManaged`` (**bool**), whether we see API partners managing ads for the user (not true if an api partner is using RO scopes only)
```
Comment (remenska) [inline src/source/release-notes.rst:125]: I'm not sure why we decided to expose it in the end :-/ it was supposed to be a temporary thing, given the implementation is based also on "proxy" data. It could be that the user is API managed but not much activity in the past (90?) days so we see no active tokens

**Review by remenska — COMMENTED** [review]
left some comments, it's your call, otherwise looks goood

File: src/source/examples/post-campaign-v5-example.rst (hunk 2 above)
Comment (twessling-icas) [reply src/source/examples/post-campaign-v5-example.rst:3]: apart from issues on our side regarding writing to multiple sources, yes

File: src/source/examples/post-campaign-v5-example.rst (hunk 3 above)
Comment (twessling-icas) [reply src/source/examples/post-campaign-v5-example.rst:20]: yes, we chose for this to make it easier for the partners to understand. I know this is not according to what we usually do, but we also had some 'invisible' versions on some endpoints just for console...

We should really get some good discussions going on API versioning :D

File: src/source/reference/get-user.rst (hunk 4 above)
Comment (twessling-icas) [reply src/source/reference/get-user.rst]: good point, wll check that out.

File: src/source/release-notes.rst (hunk 6 above)
Comment (twessling-icas) [reply src/source/release-notes.rst]: hm, we have fields called 'bidMicros', so in a way, yes it will be introduced in this version. I can make an explicit piece about that re-naming.
I'll also rephrase this section to make it clearer.

File: src/source/release-notes.rst (hunk 8 above)
Comment (twessling-icas) [reply src/source/release-notes.rst:23]: bullet lists broken in this version of sphinx/docutils 😭 I can make a separate heading, sure.

File: src/source/release-notes.rst (hunk 9 above)
Comment (twessling-icas) [reply src/source/release-notes.rst:20]: I can rephrase it in context of 'traffic shape has changed since this feature was built, it's reliability has been going down over time' ?

Comment (twessling-icas) [reply src/source/release-notes.rst]: will ask!

File: src/source/release-notes.rst (hunk 11 above)
Comment (twessling-icas) [reply src/source/release-notes.rst:125]: if we want to have only 1 user version available, we need to be able to explain all the fields. I'd also rather remove these two fields, but 
1) console needs it 
2) BNL wants to keep the functionality around `isAPiManaged` since it's about an upsell message 😭
